string output_filepath
sensor_msgs/PointCloud2 input_cloud
---
string[] segmented_mesh_filenames